### DISCUSSION TITLE
Github Desktop startup can't do anything but clone

### DISCUSSION THREAD

**Discussion by williamatwood41** [community/community#21692] [New to GitHub]
<p>Upgraded my computer to Windows 10, with a clean install.  I copied my previous Github folder to the upgraded machine, but had to re-install Github Desktop (GD).  GD discovers the project in the Github folder, but the only action it allows me to take is to clone it.  When I accept to clone it, GD complains that the foler is not empty.  What I actually want to do is pull the latest changes from the on-line repository, but I see no way to do that.</p>

**Comment by steveward** [comment, marked as answer]
<p>&gt; but the only action it allows me to take is to clone it.  When I accept to clone it, GD complains that the foler is not empty.</p>
<p>It sounds like you are trying to clone the repository instead of adding a local repository. What you will want to do is go to <code>File</code> &gt; <code>Add Local Repository</code> and navigate to your repository on disk. This will then add it to GitHub Desktop and allow you to fetch changes from the remote repository. </p>
<p>Let me know if you have any issues!</p>

**Comment by williamatwood41** [comment]
<p>Thanks.  That was the issue.</p>
